### PR TITLE
chapter_13

### PR DESCRIPTION
**Pull request by FirdausIsa**
ready

### CODE REVIEW

**Comment by FirdausIsa** [comment]
ready

File: app/assets/stylesheets/microposts.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Microposts controller here.
+// They will automatically be included in application.css.
```
Comment (anlvq-1058) [inline app/assets/stylesheets/microposts.scss:2]: please delete this files

File: test/controllers/microposts_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class MicropostsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```
Comment (anlvq-1058) [inline test/controllers/microposts_controller_test.rb:4]: please delete this file

File: test/fixtures/microposts.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (anlvq-1058) [inline test/fixtures/microposts.yml:3]: please delete this file

File: test/models/micropost_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class MicropostTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (anlvq-1058) [inline test/models/micropost_test.rb:7]: please delete this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by FirdausIsa** [comment]
ready

File: app/helpers/microposts_helper.rb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+module MicropostsHelper
```
Comment (anlvq-1058) [inline app/helpers/microposts_helper.rb:1]: please delete this file

**Review by anlvq-1058 — COMMENTED** [review]
small comment

**Comment by FirdausIsa** [comment]
ready

**Comment by FirdausIsa** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**